Sources/AWSSDKSwiftCore/AWSClient.swift

```diff
@@ -151,7 +151,7 @@ extension AWSClient {
             }
     }
 
-    public func send<Output: AWSShape>(operation operationName: String, path: String, httpMethod: String) throws -> Future<Output> {
+    public func send<Output: AWSShape>(operation operationName: String, path: String, httpMethod: String) -> Future<Output> {
 
         return signer.manageCredential().thenThrowing { _ in
```

Comment:
Great save! `thenThrowing`, renamed to [`flatMapThrowing`](https://github.com/apple/swift-nio/blob/master/Sources/NIO/EventLoopFuture.swift#L520) in NIO2, does not throw, but returns a failed future on error 🎉